app/models/notification.rb

```diff
@@ -0,0 +1,39 @@
+class Notification < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :discussion
+
+  validates :text, :url, :user, presence: true
+
+  after_create :send_email
+
+  def self.new_discussion(discussion)
+    create(
+      discussion_id: discussion.id,
+      text: I18n.t('notifications.new_discussion', author: discussion.author, discussion: discussion.title),
+      url: root_url(anchor: "#/discussion/#{discussion.id}")
```

Comment:
Yes, it was missing from the comment url

I don't know how that layer would work. At the time, I asked the front-end guys if there was a better way to do this. They told me there wasn't